README.rst

```diff
@@ -248,20 +253,47 @@ To simplify data conversion from Spark to TensorFlow, you can use Spark converte
     # make a tensorflow dataset from `converter1
     with converter1.make_tf_dataset() as dataset:
         # the `dataset` is `tf.data.Dataset` object
-        # we can train/evaluate model on `dataset`
-        # when exit the with context, the reader of the dataset will be closed
-        ...
+        # dataset transformation can be done if needed
+        dataset = dataset.map(...)
+        # we can train/evaluate model on the `dataset`
+        model.fit(dataset)
+        # when exiting the context, the reader of the dataset will be closed
 
     # delete the cached files of the dataframe.
     converter1.delete()
 
+The minimalist example below assumes the definition of a ``Net`` class and
+``train`` and ``test`` functions, included in
+`pytorch_example.py <https://github.com/uber/petastorm/blob/master/examples/mnist/pytorch_example.py>`_,
+and a Spark DataFrame containing a feature column followed by a label column.
+
+.. code-block:: python
+
+    from petastorm.spark import SparkDatasetConverter, make_spark_converter
```

Comment:
SparkDatasetConverter is not needed. Please also update the keras example.

Reply:
It is used in L274 to refer to the `PARENT_CACHE_DIR_URL_CONF`.